fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -453,6 +453,14 @@ private boolean isAuthorized(final Subject currentUser, final HttpServletRequest
                     log.debug("DELETE prohibited without {} permission", toControl);
                     return false;
                 }
+            } else if (isTxEndpoint) {
```

Comment:
I was considering not doing this so that there isn't any transaction specific logic, however the `isAuthorizedForMembershipResource` throws a NPE as both the resource and container are null. Not really a big deal but something I wanted to mention.